packages/core/src/encoding.ts

```diff
@@ -63,3 +69,41 @@ export const createLabeledSilentPaymentAddress = (
 const hrpFromNetwork = (network: Network): string => {
     return network.bech32 === 'bc' ? 'sp' : 'tsp';
 };
+
+export const parseSilentBlock = (data: Buffer): SilentBlock => {
+    const type = data.readUInt8(0);    // read 8 bits from 0th index // unsigned integer - UInt
```

Comment:
I think the method `readUInt8` is self-explanatory and doesn't warrant a comment explaining what it does.

```suggestion
    const type = data.readUInt8(0);
```